src/gfx/CMakeLists.txt

```diff
@@ -41,12 +57,22 @@ target_include_directories(gfx
 target_link_libraries(gfx
   tinygltf
   magic_enum nameof linalg
-  spdlog stb xxHash Boost::algorithm Boost::filesystem
+  spdlog stb xxHash Boost::algorithm Boost::filesystem Boost::container
```

Comment:
Do we really need `Boost::container`? there are far better alternatives on github if we need specific containers.

Reply:
I'm only using it for the allocators and memory resources:
```cpp
#include <boost/container/pmr/monotonic_buffer_resource.hpp>
#include <boost/container/pmr/memory_resource.hpp>
#include <boost/container/pmr/polymorphic_allocator.hpp>
```

Technically they should also be in the C++17 standard library, however clang seems to be slow so for now I'm using the boost implementation (still alongside the std:: containers)

For reference: https://en.cppreference.com/w/cpp/memory/polymorphic_allocator

Reply:
and why would you need polymorphic allocators anyway btw?

Reply:
I highlighted most of the usages in the first post.
Most of what I try to do can be done using cached arrays too but as soon as you start nesting stuff and having separate data for each thread this makes it a lot easier to manage I think.